### ISSUE TITLE
feat: process information

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2760]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.42.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.41.1...4.42.0) (2022-06-24)

- Release 4.42.0 [`a36d906`](https://github.com/open-wa/wa-automate-nodejs/commit/a36d9065f73a8fba3270d1ae39534efdb137bc79)
- ✨ feat: `client.getProcessStats()` #2760 [`35318bb`](https://github.com/open-wa/wa-automate-nodejs/commit/35318bbafbbfd5f8e69eec3d4c84b5cbf873b9e0)
- 🏷️ upgrade typing in line with pptr 15 breaking changes #2752 [`a996a43`](https://github.com/open-wa/wa-automate-nodejs/commit/a996a43f940662ee9c50ca19fc6720178fa3b05a)
- 🩹 patch pptr-proxy dep #2755 [`81c9c6f`](https://github.com/open-wa/wa-automate-nodejs/commit/81c9c6fd02b77206e484aec9abd43f364eff9060)
- 🩹 fix SW bypass for pptr 15 #2752 [`75c59f0`](https://github.com/open-wa/wa-automate-nodejs/commit/75c59f0672adf9a18b0139cc2b7e1118d714ed49)
- ⬆️ update to latest pptr extra deps #2752 [`72cd4c0`](https://github.com/open-wa/wa-automate-nodejs/commit/72cd4c0c302563aede3d456b08af5f234cab9653)
- 🔊 added middleware verbose logging #2758 [`db26aac`](https://github.com/open-wa/wa-automate-nodejs/commit/db26aacffa9f52230f13539a0a11e90814f8c769)
- ⬆️ update to latest puppeteer #2752 [`8d806f9`](https://github.com/open-wa/wa-automate-nodejs/commit/8d806f9b2792b52a2b8c6f690ad27534f813ae15)
- 🥚 patches update [`9a9bf74`](https://github.com/open-wa/wa-automate-nodejs/commit/9a9bf749bc6009569927f652f0c2fd1d5201c352)
- 🥚 patches update [`995b3e4`](https://github.com/open-wa/wa-automate-nodejs/commit/995b3e4fae4fe40051e13df170087f44204091ec)
- updated types-only package [`92a7b83`](https://github.com/open-wa/wa-automate-nodejs/commit/92a7b8338333900a5d61f2a0c6e99f32aed958fc)
